app/services/integrations/avalara/update_service.rb

```diff
@@ -21,8 +21,6 @@ def call
 
         integration.name = params[:name] if params.key?(:name)
         integration.code = params[:code] if params.key?(:code)
```

Comment:
Should it be possible to update the `company_code` as well?

Reply:
nope, `company_code` should not be editable per current designs